sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/state/OperatorStateMetadata.scala

```diff
@@ -24,7 +24,8 @@ import scala.reflect.ClassTag
 
 import org.apache.hadoop.conf.Configuration
 import org.apache.hadoop.fs.{FSDataInputStream, FSDataOutputStream, Path, PathFilter}
-import org.json4s.{Formats, NoTypeHints}
+import org.json4s.{Formats, JBool, JObject, NoTypeHints}
```

Comment:
Is `JObject` needed too ?

Reply:
It's currently needed because `StreamingJoinOperatorProperties` only has one field, but maybe I can just add another property in the operator properties to avoid this import altogether